k07398846/quiz1.py

```diff
@@ -4,13 +4,26 @@
 #
 
 # declare an int variable named total with the value of 0
+total = 0 
 
 # declare an array named numbers with all numbers 0 - 20
+number = [0,26,1]
```

Comment:
This is a list (array) of 3 numbers, 0, 26, and 1 

I'd like it to be 0,1,2,3,4,.....19,20